CryptoPrices/Sources/Home/Tests/HomeTests/HomeViewModelSpec.swift

```diff
@@ -120,6 +120,79 @@ final class HomeViewModelSpec: QuickSpec {
                     }
                 }
             }
+
+            describe("its fetchAllData() call") {
+
+                context("when myCoinsUseCase and trendingCoinsUseCase return success") {
+                    let returnValue = [MockCoin.single]
+                    let expectedMyCoins = [MyCoinItem(coin: MockCoin.single)]
+                    let expectedTrendinglCoins = [TrendingCoinItem(coin: MockCoin.single)]
+
+                    beforeEach {
+                        myCoinsUseCase.executeReturnValue = returnValue
+                        trendingCoinsUseCase.executeCoinIDsReturnValue = returnValue
+                        await homeViewModel.fetchAllData()
+                    }
+
+                    it("gets the correct value for myCoins") {
+                        await expect { await homeViewModel.myCoins }
+                            .to(equal(expectedMyCoins))
+                    }
+
+                    it("gets the correct value for trendingCoins") {
+                        await expect { await homeViewModel.trendingCoins }
+                            .to(equal(expectedTrendinglCoins))
+                    }
+                }
+            }
+
+            describe("its fetchAllData() call") {
+
+                context("when myCoinsUseCase returns success, trendingCoinsUseCase returns failure") {
```

Comment:
I think we can omit this `describe` and the below one and put all the contexts into the one `describe("its fetchAllData()) `call") above. What do you think?

Reply:
Oh, that's my bad 🙏  Updated in https://github.com/nimblehq/nimble-crypto-ios/pull/88/commits/6873240d834d3f2e0f2a6fde4cb33bf9f17fe8f1